ProjFS.Mac/PrjFSKextTests/KextLogMock.h

```diff
@@ -0,0 +1,10 @@
+#ifndef KextLogMock_h
+#define KextLogMock_h
+
+inline void KextLog_Error(const char* fmt, ...) {}
```

Comment:
We could do something fancier for testing logging in the future. Not something to address now.